src/easylink/pipeline_schema_constants/development.py

```diff
@@ -1,24 +1,89 @@
 from easylink.graph_components import Edge, InputSlot, OutputSlot, SlotMapping
-from easylink.step import BasicStep, HierarchicalStep, InputSlot, IOStep, LoopStep
+from easylink.step import (
+    BasicStep,
+    HierarchicalStep,
+    InputSlot,
+    IOStep,
+    LoopStep,
+    ParallelStep,
+)
 from easylink.utilities.validation_utils import validate_input_file_dummy
 
 NODES = [
-    IOStep(step_name="input_data", input_slots=[], output_slots=[OutputSlot("file1")]),
-    BasicStep(
+    IOStep(step_name="input_data", input_slots=[], output_slots=[OutputSlot("all")]),
+    ParallelStep(
         step_name="step_1",
         input_slots=[
             InputSlot(
                 name="step_1_main_input",
                 env_var="DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS",
                 validator=validate_input_file_dummy,
             ),
-            InputSlot(
-                name="step_1_secondary_input",
-                env_var="DUMMY_CONTAINER_SECONDARY_INPUT_FILE_PATHS",
-                validator=validate_input_file_dummy,
-            ),
         ],
         output_slots=[OutputSlot("step_1_main_output")],
+        split_node=HierarchicalStep(
+            "step_1",
+            input_slots=[
+                InputSlot(
+                    name="step_1_main_input",
+                    env_var="DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS",
+                    validator=validate_input_file_dummy,
+                ),
+            ],
+            output_slots=[OutputSlot("step_1_main_output")],
+            nodes=[
```

Comment:
This changed b/c you converted step 1 to a multiple step, correct? And this would be the same IOStep as a hierarchical step?

Reply:
> This changed b/c you converted step 1 to a multiple step, correct?
yep!

> And this would be the same IOStep as a hierarchical step?
I'm not sure I understand the meaning behind the question--IOStep is a node in the graph, so for any node that has edges to that IOStep, it's the "same" IOStep. The only thing I changed about the IOStep is the output slot name, since "file1" isn't really contextually meaningful.